client/components/ColorPicker.js

```diff
@@ -0,0 +1,66 @@
+export default class ColorPicker extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    generateColor() {
+        let convertRGBToHex = function(rgb) {
```

Comment:
This function is used only once. Feel free to inline it at the place it's called.